src/pages/Account/AccountGeneral.tsx

```diff
@@ -0,0 +1,167 @@
+import * as React from 'react';
+import { Button, Col, Form, FormGroup, FormFeedback, Input, Label, Row } from 'reactstrap';
+import { compose, graphql } from 'react-apollo';
+import { UPDATE_USER, User, GET_ACCOUNT_PAGE, UserField } from 'networking/users';
+import { Formik, FormikProps, FormikActions } from 'formik';
+import * as Yup from 'yup';
+import Textarea from 'components/shared/Textarea';
+import { TextareaEvent } from 'components/shared/Textarea/Textarea';
+
+interface FormValues {
+  [name: string]: boolean | string | string[] | number | object | undefined;
+}
+
+const GeneralInfoSchema = Yup.object().shape({
+  [UserField.FIRST_NAME]: Yup.string()
+    .required('Please provide your first name.'),
+  [UserField.LAST_NAME]: Yup.string()
+    .required('Please provide your last name.'),
+  [UserField.EMAIL]: Yup.string(),
+  [UserField.ABOUT]: Yup.string()
+    .required('Please fill out the About section.'),
+})
+
+const defaultValues: FormValues = {
```

Comment:
Would it be a useful simplification to do something like this?

    const defaultValues = Object.key(UserField).reduce(
      (values, key) => ({ ...values, [UserField[key]]: '' })
    );

That would reduce the number of places we'd need to update when adding a new form field, tho it only makes sense to do that if defaults will always be empty strings

Reply:
good thing to consider, though I feel each field value declared explicitly feels a bit easier to read.

If other forms do include numbers/arrays/booleans, it would be easier to add / adjust at first glance.